qa/L0_backend_python/model_control/model_control_test.py

```diff
@@ -82,6 +82,23 @@ def test_model_reload(self):
                 self.assertFalse(client.is_model_ready(model_name))
                 self.assertFalse(client.is_model_ready(ensemble_model_name))
 
+    def test_faulty_model_load(self):
+        working_model_name = "identity_fp32"
+        faulty_model_name = "auto_complete_error"
+        with httpclient.InferenceServerClient(f"{_tritonserver_ipaddr}:8000") as client:
+            # Load a correct model
+            client.load_model(working_model_name)
```

Comment:
Can you add a comment why a working model is required to reproduce the issue?

Reply:
I am still trying to figure this out.
I think we can merge this Test for 24.05 and meanwhile I will continue my investigation.
We have already merged the python backend change.